test/unit/ids.js

```diff
@@ -16,26 +17,31 @@ const mockDb = (idFilterLogicFn) => {
 };
 
 exports.tearDown = callback => {
-    testUtils.restore([
-        Math.random
-    ]);
+  testUtils.restore([
+    Math.random
+  ]);
 
-    callback();
+  callback();
 };
 
-exports['generates an id of the given length'] = function(test) {
-    [1,2,3,4,5,6,7,8,9,10].forEach(function(l) {
-        test.equal(ids._generate(l).length, l);
-    });
+exports['generates an id of the given length'] = test => {
+  [1, 2, 3, 4, 5, 6, 7, 8, 9, 10].forEach(l =>
+    test.equal(ids._generate(l).length, l));
 
-    test.done();
+  test.done();
 };
 
-exports['ids can start with 0, will be correct length'] = function(test) {
-    sinon.stub(Math, 'random').returns(0.00001);
+exports['ids can start with 0, will be correct length'] = test => {
+  sinon.stub(Math, 'random').returns(0.00001);
 
-    test.equal(ids._generate(5), '00000');
-    test.done();
+  test.equal(ids._generate(5), '00000');
+  test.done();
+};
+
+exports['ids are "always" the length they should be'] = test => {
+  test.ok(
+    jsc.checkForall(jsc.integer(5, 13), i => ids._generate(i).length === i));
```

Comment:
Ok, so I don't understand what it's doing then because I'm unfamiliar with QuickCheck. It looks to me like you're testing the same thing as the `generates an id of the given length` test.  If you want to enlighten me re QuickCheck that's cool, otherwise I'll read up on it soon, thanks!